### DISCUSSION TITLE
[2215430]对马岛之魂导演剪辑版Ghost of Tsushima DIRECTOR'S CUT {5.16本体}

### DISCUSSION THREAD

**Discussion by yasilelei** [pjy612/SteamManifestCache#664] [Share]
### Steam 游戏ID

`appid:2215430`

### 密钥列表

```hash
2215431;dfc6d3958470277318949e0773165b9ce5d26f6ae73e2d37b9edfe4b934067a1
2215432;b218c016d1b02dd7d4722fe5d6bcf3d44d8a8c978f493678df8a98d855e74c8a
2215434;d712c2f977a832063ecdee3943236ccfdfc386c19d05a30c85a880273ab6f110
2215438;7700f2391ea9b7bdeae2d69c20788091cad2d095c83a4c78f0d2ba27673b75d3
```

### 清单附件

[2215431_7464135075330126981.zip](https://github.com/pjy612/SteamManifestCache/files/15345469/2215431_7464135075330126981.zip)

### 备注

code:1D8408BAD58466D8
email:1874386614@qq.com

**Comment by Anonymous-1901** [comment]
Hi,

New to this, can i get some help with creating the lua script?
